src/OrchardCore.Modules/OrchardCore.Html/GraphQL/HtmlBodyInputObjectType.cs

```diff
@@ -0,0 +1,15 @@
+using OrchardCore.Apis.GraphQL.Queries;
+using OrchardCore.Html.Model;
+
+namespace OrchardCore.Html.GraphQL
+{
+    public class HtmlBodyInputObjectType : QueryArgumentObjectGraphType<HtmlBodyPart>
```

Comment:
I couldn't find anything for MarkdownPart. Is it missing? If so should we do it for all the parts?
I looked for it because I think we should be able expose the html transformation of the markdown property, if requested in the query.

Reply:
I agree - Added